addons/calendar/models/calendar.py

```diff
@@ -1355,7 +1386,19 @@ def _name_search(self, name='', args=None, operator='ilike', limit=100, name_get
         return super(Meeting, self)._name_search(name=name, args=args, operator=operator, limit=limit, name_get_uid=name_get_uid)
 
     @api.multi
+    def update_mail_activity(self, values):
+        if values.get('name'):
+            self.activity_id.summary = values['name']
+        if values.get('description'):
+            self.activity_id.note = values['description']
+        if values.get('start'):
+            self.activity_id.date_deadline = fields.Datetime.from_string(values['start']).date()
+
+    @api.multi
     def write(self, values):
+        # update mail.activity if date or time is change
```

Comment:
this comment isn't in sync with what it does, I'd remove it. (also updates name, description..)